### PR TITLE
[FEATURE] MS-70: Admins - Positions

### PR DESCRIPTION
**Pull request by juanandres7698**

### CODE REVIEW

File: src/Components/shared/ListItem/index.jsx [hunk 1]
```diff
@@ -98,16 +98,23 @@ function ListItem({
...
             <span className="material-icons-outlined">clear</span>
           </button>
         )}
+        {/*
         {statusSession !== 'cancelled' && viewMore && (
           <button className="view-btn" onClick={viewElement}>
             <span className="material-icons-outlined">search</span>
           </button>
         )}
+        */}
```
Comment (EugenioPonzone) [inline src/Components/shared/ListItem/index.jsx]: why did you comment this?

File: src/Components/shared/ListItem/index.jsx [hunk 2]
```diff
@@ -98,16 +98,23 @@ function ListItem({
             </button>
           </Link>
         )}
+        {viewMore && (
+          <button className="view-btn" onClick={viewElement}>
+            <span className="material-icons-outlined">search</span>
+          </button>
+        )}
```
Comment (EugenioPonzone) [inline src/Components/shared/ListItem/index.jsx]: and why did you add this other one?

File: src/Components/shared/ListItem/index.jsx (hunk 1 above)
Comment (Sabrucita) [inline src/Components/shared/ListItem/index.jsx:117]: Why did you comment this lines? 
statusSession !== 'cancelled' is used in other screens

**Review by Sabrucita — APPROVED** [review]

**Review by EugenioPonzone — APPROVED** [review]
